src/Modal/ModalCard.js

```diff
@@ -0,0 +1,64 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import css, { resolve } from 'styled-jsx/css'
+import cx from 'classnames'
+
+import { Card } from '../Card'
+
+const outerSpacing = 24
+
+const cardBoxStyle = resolve`
+    .modal-card {
+        height: auto;
+        left: 50%;
+        max-width: calc(100vw - ${2 * outerSpacing}px);
+        max-height: calc(100vh - ${2 * outerSpacing}px);
+        position: absolute;
+        top: 50%;
+        transform: translate(-50%, -50%);
+    }
+
+    .modal-card.scrollable {
+        height: 100%;
+    }
+
+    .size-small { width: 400px; }
+    .size-medium { width: 600px; }
+    .size-large { width: 800px; }
+`
+
+export const ModalCard = ({ children, scrollable, size }) => {
+    console.log('scrollable', scrollable)
+    return (
+        <Card
+            className={cx(
+                cardBoxStyle.className,
+                'modal-card',
+                `size-${size}`,
+                { scrollable }
+            )}
+        >
+            <div
+                style={{
+                    height: '100%',
+                    display: 'flex',
+                    flexDirection: 'column',
+                }}
+            >
+                {children}
+            </div>
+            {cardBoxStyle.styles}
+        </Card>
+    )
+}
+
+ModalCard.propTypes = {
+    // Can be Modal.Title; Must be Modal.Content and Modal.Actions
+    children: PropTypes.arrayOf(PropTypes.element).isRequired,
+    size: PropTypes.oneOf(['small', 'medium', 'large']),
```

Comment:
Default to medium, and use `small` and `large` as boolean props to match the API for other components. Check the Button for example.